configure.ac

```diff
@@ -1,4 +1,5 @@
-AC_INIT([logrotate],[3.11.0])
+AC_INIT([logrotate],[m4_esyscmd([build-aux/git-version-gen .tarball-version])],
+[logrotate-owner@fedoraproject.org],[],[https://github.com/logrotate/logrotate])
```

Comment:
Where did you get the e-mail address from?  It is an alias for Fedora package maintainer(s).  It should not be advertised as a contact to logrotate upstream...